src/main/webapp/app/shared/sidebar/conversation-options/conversation-options.component.ts

```diff
@@ -79,11 +79,33 @@ export class ConversationOptionsComponent implements OnInit, OnDestroy {
 
     onHiddenClicked(event: MouseEvent) {
         event.stopPropagation();
+        if (!this.course.id || !this.conversation.id) return;
+
+        // If the conversation is in favorites and is being hidden, it is also removed from favorites
```

Comment:
In my opinion we can remove this comment as it is not really describing something complex in the method